<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Produkty</class>
 <widget class="QDialog" name="Produkty">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1000</width>
    <height>700</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Ignored" vsizetype="Ignored">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>1000</width>
    <height>700</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>1000</width>
    <height>700</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="autoFillBackground">
   <bool>false</bool>
  </property>
  <property name="styleSheet">
   <string notr="true">#Produkty
{
background-color:rgb(0, 170, 255);
}
</string>
  </property>
  <widget class="QTableView" name="tableView">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>40</y>
     <width>461</width>
     <height>221</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">#tableView{
background-color:rgb(170, 0, 255);
}</string>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <property name="editTriggers">
    <set>QAbstractItemView::NoEditTriggers</set>
   </property>
  </widget>
  <widget class="QLabel" name="ProduktyLabel">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>0</y>
     <width>361</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>500</y>
     <width>811</width>
     <height>191</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>20</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">#groupBox{
background-color:rgb(170, 0, 255);
}</string>
   </property>
   <property name="title">
    <string>Summary</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignHCenter|Qt::AlignTop</set>
   </property>
   <widget class="QLabel" name="CenaLabel2">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>40</y>
      <width>131</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>11</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="CenaLabel1">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>80</y>
      <width>131</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>11</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QTableView" name="tableView_2">
    <property name="geometry">
     <rect>
      <x>570</x>
      <y>40</y>
      <width>31</width>
      <height>21</height>
     </rect>
    </property>
    <attribute name="horizontalHeaderVisible">
     <bool>false</bool>
    </attribute>
    <attribute name="verticalHeaderVisible">
     <bool>false</bool>
    </attribute>
   </widget>
   <widget class="QTableView" name="tableView_3">
    <property name="geometry">
     <rect>
      <x>570</x>
      <y>80</y>
      <width>31</width>
      <height>21</height>
     </rect>
    </property>
    <attribute name="horizontalHeaderVisible">
     <bool>false</bool>
    </attribute>
    <attribute name="verticalHeaderVisible">
     <bool>false</bool>
    </attribute>
   </widget>
   <widget class="QPushButton" name="ZamowButton">
    <property name="geometry">
     <rect>
      <x>540</x>
      <y>140</y>
      <width>181</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">#ZamowButton
{


background-color :rgb(0, 170, 255);
color : black;
border-style: outset;
}
</string>
    </property>
    <property name="text">
     <string>Add to cart</string>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox_2">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>30</y>
      <width>261</width>
      <height>161</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="title">
     <string>Order details</string>
    </property>
    <widget class="QLabel" name="label_4">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>171</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>The total cost of order:</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>190</x>
       <y>20</y>
       <width>41</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QPushButton" name="ZamowButtn2">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>120</y>
       <width>171</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">#ZamowButtn2
{

background-color :rgb(0, 170, 255);
color : black;
border-style: outset;
}
</string>
     </property>
     <property name="text">
      <string>Order</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="AdresEdit">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>80</y>
       <width>101</width>
       <height>16</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_5">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>80</y>
       <width>61</width>
       <height>20</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>Address:</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="Nazwiskoedit">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>60</y>
       <width>101</width>
       <height>16</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="ImieEdit">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>40</y>
       <width>101</width>
       <height>16</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_7">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>40</y>
       <width>47</width>
       <height>13</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>Name:</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_8">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>60</y>
       <width>61</width>
       <height>16</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>Surname:</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_9">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>100</y>
       <width>61</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>City:</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="MiastoEdit">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>100</y>
       <width>101</width>
       <height>16</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="Podsumowanie_3">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>-230</y>
       <width>771</width>
       <height>241</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>18</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="layoutDirection">
      <enum>Qt::LeftToRight</enum>
     </property>
     <property name="inputMethodHints">
      <set>Qt::ImhNone</set>
     </property>
     <property name="lineWidth">
      <number>1</number>
     </property>
     <property name="midLineWidth">
      <number>0</number>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="textFormat">
      <enum>Qt::AutoText</enum>
     </property>
     <property name="scaledContents">
      <bool>true</bool>
     </property>
     <property name="wordWrap">
      <bool>true</bool>
     </property>
     <property name="margin">
      <number>2</number>
     </property>
    </widget>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>110</y>
      <width>71</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>11</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Amount:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="IloscEdit">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>130</y>
      <width>31</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="WyszukajProdukt">
   <property name="geometry">
    <rect>
     <x>550</x>
     <y>40</y>
     <width>351</width>
     <height>221</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>16</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="whatsThis">
    <string extracomment="test"/>
   </property>
   <property name="styleSheet">
    <string notr="true">#WyszukajProdukt{
background-color:rgb(170, 0, 255);
}</string>
   </property>
   <property name="title">
    <string>Search</string>
   </property>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>70</y>
      <width>141</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Product name :</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>110</y>
      <width>161</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Producer:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="WyszukajNazwa">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>70</y>
      <width>161</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="WyszkuajFirma">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>110</y>
      <width>161</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="WyszkajButton">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>150</y>
      <width>191</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">#WyszkajButton{

background-color :rgb(0, 170, 255);
color : black;
border-style: outset;
}</string>
    </property>
    <property name="text">
     <string>Check availability and price</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_11">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>30</y>
      <width>131</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">#label_11{
color : rgb(255, 255, 0)
}</string>
    </property>
    <property name="text">
     <string>City:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="WyszukajMiasto">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>30</y>
      <width>161</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QLabel" name="label_10">
   <property name="geometry">
    <rect>
     <x>390</x>
     <y>200</y>
     <width>47</width>
     <height>13</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="Podsumowanie">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>270</y>
     <width>881</width>
     <height>131</height>
    </rect>
   </property>
   <property name="maximumSize">
    <size>
     <width>16777215</width>
     <height>177</height>
    </size>
   </property>
   <property name="font">
    <font>
     <pointsize>16</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="layoutDirection">
    <enum>Qt::LeftToRight</enum>
   </property>
   <property name="inputMethodHints">
    <set>Qt::ImhNone</set>
   </property>
   <property name="lineWidth">
    <number>1</number>
   </property>
   <property name="midLineWidth">
    <number>0</number>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="textFormat">
    <enum>Qt::AutoText</enum>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
   <property name="wordWrap">
    <bool>true</bool>
   </property>
   <property name="margin">
    <number>2</number>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
